learning/tour-of-beam/frontend/lib/cache/content_tree.dart

```diff
@@ -18,21 +18,17 @@
 
 import 'dart:async';
 
-import 'package:flutter/widgets.dart';
-
 import '../models/content_tree.dart';
-import '../repositories/client/client.dart';
+import 'cache.dart';
 
-class ContentTreeCache extends ChangeNotifier {
-  final TobClient client;
+class ContentTreeCache extends Cache {
```

Comment:
What does a ContentTreeCache cache?  Would code comments help explain this?

Reply:
I've added a comment to the `Cache` class.